index.html

```diff
@@ -5,21 +5,44 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Ercan's playground</title>
+    <link rel="stylesheet" href="./styles.css" />
     </head>
  <style>
-    h1  {
-        text-align: center;
-    }
    </style>
   <body>
     <h1>Welcome to my website
```

Comment:
You used the h1 for quite a big part of your page. I would advice to really only use the h1 tag a heading and not include links to your github/linkedIn etc. in it.

Rest looks good to me :)

Reply:
oh wow, i didnt notice that i put the links into the h1 section, gonna change it now, thanks for the advice!